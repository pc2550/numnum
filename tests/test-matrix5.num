
int main(){
	float [1] a;
	float [2][1] b;
	float [2][1] c;

    a[0] = 4.0; 
    b[0][0] = 3.0;
    b[1][0] = 5.0;
    c[0][0] = 1.0;
    c[1][0] = 1.0;


    printfl(c[0][0]);

    bc_add(a, b, c);

    printfl(c[0][0]);
    printfl(c[1][0]);
    
    bc_sub(a, b, c);

    printfl(c[0][0]);
    printfl(c[1][0]);

    bc_mul(a, b, c);

    printfl(c[0][0]);
    printfl(c[1][0]);

	 bc_div(a, b, c);

    printfl(c[0][0]);
    printfl(c[1][0]);
return 0;
}
