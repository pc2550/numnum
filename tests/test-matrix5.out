1.000000
7.000000
9.000000
1.000000
-1.000000
12.000000
20.000000
1.333333
0.800000
